src/features/p2-RegPage/RegPage.tsx

```diff
@@ -9,17 +9,15 @@ import {Preloader} from "../../b1-ui/common/preloader/Preloader";
 
 export const RegPage = () => {
 
-    const notification = useSelector<AppStateType, string>(state => state.regPage.notification)
-    const {isRegistrated, inProgress} = useSelector<AppStateType, InitialStateType>(state => state.regPage)
-
+    const notification = useSelector<AppStateType, string>(state => state.regPage.notification);
+    const {isRegistrated, inProgress} = useSelector<AppStateType, InitialStateType>(state => state.regPage);
 
     const dispatch = useDispatch();
 
     const submitData = () => {
         dispatch(sendUserData(email, password))
     };
 
-
```

Comment:
could you plz delete one more string here?

Reply:
I rewrite it.